tests/baselines/reference/umdNamespaceMergedWithGlobalAugmentationIsNotCircular.types

```diff
@@ -10,10 +10,10 @@ export { };
 
 === tests/cases/compiler/module.d.ts ===
 export = React;
->React : typeof React
+>React : typeof import("tests/cases/compiler/module")

```

Comment:
The UMD namespace declaration makes us prefer this path to this reference, since the global `React` name is "poisoned" in the module scope.

Still tho - it's a module declaring itself as a global namespace and exporting that as itself - that's so uselessly circular, it doesn't make sense whichever name you choose.